README.rst

```diff
@@ -17,7 +17,11 @@
 coala supports popular programming languages including **Python**, **C/C++**, **Java**, **JavaScript**, **CSS**, and `several others <http://coala.io/#!/languages>`_ out of the box.
 
 
-`Install coala <http://coala.io/#!/home#installation>`_ | `Get Involved <http://coala.io/#/getinvolved>`_ | `Chat <https://coala.io/chat>`_ | `Roadmap <https://projects.coala.io>`_ | `Blog <http://blog.coala.io>`_ | `Twitter <https://www.twitter.com/coala_io>`_
+<<<<<<< HEAD
```

Comment:
Awesome work @2knal. However, it looks like a merge-conflict resolution was interrupted in between. Please fix this by choosing the correct block. Amend the commit (`git commit --amend`) and force push the changes!

Reply:
Sorry for the mistake,
Is it fine now?